demo/index.html

```diff
@@ -16,10 +16,39 @@
 
     <demo-navigation></demo-navigation>
 
-    <h3>Sample example</h3>
+    <h3>Default button</h3>
     <demo-snippet>
       <template>
-        <vaadin-button></vaadin-button>
+        <vaadin-button>Vaadin Button</vaadin-button>
+        vs
+        <button>Native button</button>
+      </template>
+    </demo-snippet>
+
+    <h3>Button with autofocus</h3>
+    <demo-snippet>
+      <template>
+        <vaadin-button autofocus>
+          <img src="vaadin-logo-small.png">
```

Comment:
> > The image file is an old logo, don’t use.

> It's the current logo on vaadin.com

The new branding is official and ready to use, but the website can not be quickly migrated. It’s better to use the new branding for the new things we start nowadays.

> > Please also consider high-DPI screens support.

> I'm not sure about this, what exactly should I change?

For an MVP, pick a file with 2× pixel density, so that it looks nice.